media-accelerator/get-started/get-started-sdk.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'SDK quickstart'
+sidebar_position: 1
+type: docs
+description: >
+  Rapidly develop and easily enhance IoT solutions with audio communication and face-to-face interaction.
+---
+
+import GetStartedSDK from '@docs/shared/media-accelerator/get-started/_get-started-sdk.mdx';
+
+export const toc = [{}];
+
+<GetStartedSDK  product="Media Accelerator" company="Agora" sdk="RTSA SDK" client="app" />
```

Comment:
Media Accelerator SDK. 

Don't we have global variables for this?

Reply:
Removed.